sympy/matrices/matrices.py

```diff
@@ -2861,7 +2861,7 @@ def gauss_jordan_solve(self, b, freevar=False):
         =======
 
         x : Matrix
-            The matrix that will satisfy Ax = B.  Will have as many rows as
+            The matrix that will satisfy ``Ax = B``.  Will have as many rows as
```

Comment:
Generally only valid Python code should go in a code block. So this and other similar instances should either be left alone or use a math environment.